scripts/tests/test-ci-all.sh

```diff
@@ -182,7 +182,16 @@ if [[ "$num_versions" == "0" ]]; then
 else
   # precompile binaries
   binaries=( "fedimintd" "fedimint-cli" "gateway-cli" "gatewayd" )
-  parallel nix_build_binary_for_version "{1}" "{2}" ::: "${binaries[@]}" ::: "${tagged_versions[@]}"
```

Comment:
Out of curiosity, did you compare the performance of removing `parallel`? It might be faster in sequence due to resource contention (originally flagged by @elsirion in https://github.com/fedimint/fedimint/pull/4413#discussion_r1501815988).

Reply:
If `nix build` is not contended all of this is rather instant.

Reply:
Yep, that makes sense. I was curious about the initial build that we'll do for all versions (https://github.com/fedimint/fedimint/pull/5144/files#diff-e5c7cc37f7292b713d50077a8badf27398452732e2042c0bb9a57b4d8346fa81R191)